core/code/search.js

```diff
@@ -393,11 +393,57 @@ addHook('search', function(query) {
       query.addResult(result);
     });
   }
-  
+
   // Bounded search allows amenity-only searches (e.g. "amenity=toilet") via special phrases
   // http://wiki.openstreetmap.org/wiki/Nominatim/Special_Phrases/EN
   var bounded = '&bounded=1';
-  
+
   $.getJSON(NOMINATIM + encodeURIComponent(query.term) + viewbox + bounded, onQueryResult.bind(null, true));
 });
 
+// search for guid
+window.search.addGuidResult = function (query, data, guid) {
```

Comment:
Yup, this method is duplication of code around line 270...
The only difference that it uses new iitc apis..

Removed duplication and added machina support (as per @le-jeu comment) here - https://github.com/MysticJay/ingress-intel-total-conversion/pull/12

Reply:
merged proposal from @xscreach